reference/3.0/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -282,14 +283,29 @@ Param
 )
 ```
 
+> [!NOTE]
```

Comment:
Make this NOTE block match the one from about_Parameters.

Reply:
Updated and synchronized

Reply:
Also synced all of the about_functions_advanced_parameter files.